python/semantic_kernel/connectors/ai/azure_ai_inference/services/azure_ai_inference_chat_completion.py

```diff
@@ -1,24 +1,27 @@
 # Copyright (c) Microsoft. All rights reserved.
 
+import asyncio
 import logging
+import sys
 from collections.abc import AsyncGenerator
+from functools import reduce
 from typing import Any
 
+from semantic_kernel.connectors.ai.function_calling_utils import update_settings_from_function_call_configuration
```

Comment:
let's move this down with the other imports, the `if sys...` line throws off the regular import sorting

Reply:
Good catch!